demisto_sdk/commands/common/hook_validations/conf_json.py

```diff
@@ -75,3 +76,47 @@ def is_test_in_conf_json(self, file_id):
         if self.handle_error(error_message, error_code, file_path=CONF_PATH):
             return False
         return True
+
+    def is_valid_file_in_conf_json(self, current_file, file_type, file_path):
+        """Check if the file is valid in the conf.json"""
+        entity_id = _get_file_id(file_type.value, current_file)
+        if file_type == FileType.INTEGRATION:
+            return self.integration_has_unskipped_test_playbook(current_file, entity_id, file_path)
+        if file_type == FileType.SCRIPT:
+            return self.has_unskipped_test_playbook(current_file=current_file,
+                                                    entity_id=entity_id,
+                                                    file_path=file_path)
+        return True
+
+    def has_unskipped_test_playbook(self, current_file, entity_id, file_path, test_playbook_ids: list = []):
+        """Check if the content entity has at least one unskipped test playbook."""
+        test_playbooks_unskip_status = {}
+        all_test_playbook_ids = test_playbook_ids.copy()
+        skipped_tests = self.conf_data.get('skipped_tests', {})
+
+        if type(current_file.get('tests')) is list:
+            all_test_playbook_ids.extend(current_file.get('tests', []))
+
+        for test_playbook_id in set(all_test_playbook_ids):
+            if skipped_tests and test_playbook_id in skipped_tests:
```

Comment:
why do you need to first part of the condition?

Reply:
if the skipped_tests section is None in the conf_json instead of an empty dict, something like {'skipped_tests': None}